core/sql/sqlcomp/DefaultConstants.h

```diff
@@ -2585,6 +2585,7 @@ enum DefaultConstants
   SCRATCH_MGMT_OPTION,
   SCRATCH_PREALLOCATE_EXTENTS,
   SCRATCH_IO_BLOCKSIZE_SORT,
+  SCRATCH_IO_BLOCKSIZE_SORT_MAX,
```

Comment:
Since we have a new configurable MAX, can the previous CQD be removed and be made a constant in the code ?